### DISCUSSION TITLE
Indi asi camera framing

### DISCUSSION THREAD

**Discussion by tbessell** [mworion/MountWizzard4#238] [Q&A]
Hi,
How do I get MountWizzard4 to honor the framing and size parameters that I set in kstars and Indi. I have entered the information in the Indi control panel and in ekos and mountwizzard still captures a full frame image.

indi v1.9.2, kstars 3.5.5, zwo asi6200mm pro running manjaro on 8G raspberry pi.

Thank you for your help,
Tim

**Comment by mworion** [comment]
Hi Tim,
MW4 does not use the Frame and size parameters of the imaging app, you set this in imaging tab for the modeling process in MW4. The reason is simple: most users normally choose different settings for modeling / analyse than for imaging itself. For modeling e.g. 1500 pix x/y is sufficient, you might do bin or subframe. The same is valid for gain or offset in case of CMOS cam. Your imaging application should stay in the mode for imaging. 
Michel

**Reply by tbessell** [reply]
I agree with your reasoning and that sounds logical. I looked in the imaging tab for a frame setting. I saw sub frame, but didn't see how to set framing. I must have missed it, I'll look again.

Thank you for the speedy reply !

**Reply by tbessell** [reply]
I found a screen shot of the imaging tab online and see where frame size is set in pixels. Suppose I want to center my framing in the middle of the cmos chip. I would also need to specify an X/Y position or will mountwizzard do that automagically :) ?

Thanks so much, Tim

**Reply by tbessell** [reply]
I checked the image tab on my running telescope setup. I see where framing is set, but I cannot change the settings. They seem to be read only.

Thank you again for your help,
Tim

**Reply by mworion** [reply]
Hi Tim,
I just made a quick test with the ccd simulator. And frame could be used. I check if the driver offers this property to be set remotely. Unfortunately sometimes not all internal EKOS properties are offered remotely.

![Bildschirmfoto 2022-05-18 um 17 56 28](https://user-images.githubusercontent.com/6632980/169088342-ed35c214-fe47-4af0-9561-ebf8e92d29b7.png)

You just set it in percent of the overall picture. And it's centered automatically.
Michel

**Reply by tbessell** [reply]
Yes, that seems to be the case with my setup. I am able to make a change using 'Subframe %' in daytime bench testing, but test it under actual night sky conditions. 

Thank you again for the great software and your support,
Tim

**Reply by mworion** [reply]
Great to hear. Clear Skies !
